src/EnergyPlus/TarcogShading.cc

```diff
@@ -1036,7 +1036,12 @@ namespace TarcogShading {
                 H0 = (dens2 * cp2 * s * speed) / (4.0 * hc + 8.0 * speed);
 
                 P = -H / H0;
-                beta = std::pow(e, P);
+                if (P < -700.0) {
+                    beta = 0.0;
+                }
+                else {
+                    beta = std::pow(e, P);
```

Comment:
@lgu1234 @jasondegraw This bug fix work stumbled over an underflow in TacogShading. Did you agree on a plan for exp underflow protection? I'll change this to match.

Reply:
I don't recall if we ever actually agreed on what to do over on the AFN side.  @jasondegraw's [comment](https://github.com/NREL/EnergyPlus/issues/7233#issuecomment-474853258) indicates he has made some progress but we still need resolution.

Reply:
@Myoldmopar @mjwitte Oops, missed this. I did make progress, @lgu1234 believe we have a solution, but the problem is really testing. Let's talk this week.